app/controllers/content/index.js

```diff
@@ -103,10 +111,10 @@ export default Controller.extend(Analytics, {
         return false;
     }),
 
-    showStatusBanner: computed('model.{provider.reviewsWorkflow,reviewsState}', 'userIsContrib', 'node.public', function() {
+    showStatusBanner: computed('model.{public,provider.reviewsWorkflow,reviewsState}', 'userIsContrib', function() {
         return (
             this.get('model.provider.reviewsWorkflow')
-            && this.get('node.public')
+            && this.get('model.public')
```

Comment:
Do not show the reviews banner if the preprint is private.  Preprints can still be private w/ legacy and spam preprints.